nix_update/update.py

```diff
@@ -82,24 +82,28 @@ def nix_prefetch(cmd: List[str]) -> str:
 
 
 def update_src_hash(opts: Options, filename: str, current_hash: str) -> None:
-    target_hash = nix_prefetch([f"(import {opts.import_path} {{}}).{opts.attribute}"])
+    target_hash = nix_prefetch(
+        [
+            f"(import {opts.import_path} {{ config.checkMeta = false; }}).{opts.attribute}"
```

Comment:
Why is checkMeta an issue in the first place?

Reply:
How does it break your usecase? I solely use it for nixpkgs and whenever I run the prefetch on any packages it breaks because some packages uses ``inherit (src.meta) homepage``.

Reply:
```
nix-prefetch '{ sha256 }: (import ./. { config.checkMeta = true; }).lab.go-modules.overrideAttrs (_: { vendorSha256 = sha256; })'
error: attribute 'homepage' missing, at /home/hotpi/src/nixpkgs5/pkgs/tools/compression/brotli/default.nix:55:21
(use '--show-trace' to show detailed location information)
```

Reply:
I update also https://github.com/Mic92/nur-packages/ with it and potentially other folks do too.

Reply:
I think one can check if the file accepts a `config` parameter in which case it could be passed: https://github.com/nix-community/NUR/blob/f1aa60334945333dacfe32b95d45dd70c225b094/lib/evalRepo.nix#L26

Reply:
```suggestion
            f"(import {opts.import_path} (if (builtins.hasAttr "config" (builtins.functionArgs (import {opts.import_path}))) then { config.checkMeta = false; } else {})).{opts.attribute}"
```